cli/tests/tobs-cli-tests/install_backup_enabled_test.go

```diff
@@ -1,83 +0,0 @@
-package tobs_cli_tests
-
-import (
-	"os"
-	"testing"
-
-	test_utils "github.com/timescale/tobs/cli/tests/test-utils"
-)
-
-type backupDetails struct {
-	key   string
-	value string
-}
-
-func testBackUpEnabledInstallation(t *testing.T) {
```

Comment:
This is only needed when upgrading from 0.2.2. In recent versions this is obsolete.